sdk/go/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/adrg/xdg v0.4.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/iancoleman/strcase v0.2.0
+	github.com/moby/buildkit v0.10.5
```

Comment:
noooo it's back

Reply:
OMG good catch haha, it's cause I used the library they have for creating a random id (I think out of muscle memory from when I worked on that codebase more).

Will fix, can just use stdlib to get a random id.

Reply:
We should almost have a linter rule that fails if buildkit ends up in `go.mod`

FWIW -- I think this would have worked since the identity package is self contained (I don't see docker in go.mod/sum). Better safe than sorry though